### PR TITLE
Multi-retargeting support (together)

### PR DESCRIPTION
**Pull request by angbpy**

### CODE REVIEW

File: includes/api/save-abandonment-data.php [hunk 1]
```diff
@@ -80,10 +94,10 @@ function saveCartAbandonmentData(WP_REST_Request $request)
         //Insert data for tracking started checkout.
         $eventData = wck_build_cart_data($cart);
         if (empty($eventData['$extra']['Items'])) {
-            $response['status']  = false;
-            $response['message'] = 'cart item not exist in kalviyo';
-            $statusCode          = 400;
-            return new WP_REST_Response($response, $statusCode);
+            $response['status']    = false;
+            $response['message']   = 'cart item does not exist in klaviyo';
```
Comment (poojarazor) [inline includes/api/save-abandonment-data.php:98]: Are we using  $response['message'] somewhere?

File: includes/api/save-abandonment-data.php [hunk 2]
```diff
@@ -80,10 +94,10 @@ function saveCartAbandonmentData(WP_REST_Request $request)
...
+            $statusCode            = 400;
+            $result['response']    = $result['response'].PHP_EOL."\nFailed to insert data for Klaviyo plugin";
```
[the remark this answers is not in the document]
Comment (angbpy) [reply includes/api/save-abandonment-data.php:100]: Removed.

File: includes/api/save-abandonment-data.php (hunk 1 above)
Comment (angbpy) [reply includes/api/save-abandonment-data.php:98]: Removing it and updating reason in the result['response']

Comment (angbpy) [reply includes/api/save-abandonment-data.php:98]: Appending to result['response']

File: includes/api/save-abandonment-data.php [hunk 3]
```diff
@@ -99,36 +113,42 @@ function saveCartAbandonmentData(WP_REST_Request $request)
...
         //To verify whether the email id is already exist on WordPress
         if (email_exists($razorpayData['customer_details']['email'])) {
-            $response['status']  = false;
-            $response['message'] = 'For Register user we can not insert data';
-            $statusCode          = 400;
-            return new WP_REST_Response($response, $statusCode);
+            $response['status']    = false;
+            $response['message']   = 'For Register user we can not insert data';
+            $statusCode            = 400;
+            $result['response']    = $result['response'].PHP_EOL."Failed to insert data for Abandonment Cart Lite plugin";
```
[the remark this answers is not in the document]
Comment (angbpy) [reply includes/api/save-abandonment-data.php:128]: added "for registered user" to response message

**Review by poojarazor — APPROVED** [review]

**Pull request merged**